### ISSUE TITLE
VRM runtime loading with 0.94 is 23% slower than 0.66

### ISSUE THREAD

**Issue by emilianavt** [vrm-c/UniVRM#1495]
## Environments (please complete the following information):

 - UniVRM version: `0.94.0`
 - Unity version: `Unity-2019.4.31f1`
 - OS: `Windows 8.1`

## Describe the bug

Runtime loading a VRM file with UniVRM 0.94 is 23% slower than with version 0.66 for large files. I did not measure it with 0.89, but it felt slower as well.

The following code was used for loading in 0.94:

```cs
using System.IO;
using VRM;
using UniGLTF;
using UnityEngine;

public class TimeLoad : MonoBehaviour
{
    void Start() {
        for (int i = 0; i < 10; i++) {
            var watch = System.Diagnostics.Stopwatch.StartNew();
            GltfData data = new GlbFileParser("Model.vrm").Parse();
            VRMData vrmdata = new VRMData(data);
            var instance = new VRMImporterContext(vrmdata).Load();
            Debug.Log(watch.ElapsedMilliseconds/1000f);
        }
    }
}
```

The following code was used with 0.66:
```cs
using System.IO;
using VRM;
using UnityEngine;

public class TimeLoad : MonoBehaviour
{
    void Start() {
        for (int i = 0; i < 10; i++) {
            var watch = System.Diagnostics.Stopwatch.StartNew();
            byte[] bytes = File.ReadAllBytes("Model.vrm");
            var context = new VRMImporterContext();
            context.ParseGlb(bytes);
            context.Load();
            Debug.Log(watch.ElapsedMilliseconds/1000f);
        }
    }
}
```

With an 8.7MB model like [Vita](https://www.vseeface.icu/Vita.vrm) the difference in speed is 6% to 30%, depending on which measurements are ignored as outliers:

0.94
![Timing0_94](https://user-images.githubusercontent.com/38952746/152178694-9274b197-759f-4b52-a5e4-2b92eb698a91.png)

0.66
![Timing0_66](https://user-images.githubusercontent.com/38952746/152178709-5d302e00-9b03-4cca-85e8-33ffb2fb155b.png)

For a larger 131.5MB model, the difference in loading time becomes very clear:

0.94
![EMTiming0_94](https://user-images.githubusercontent.com/38952746/152178837-35e56382-1f79-47a1-80c9-de2eec16140c.png)

0.66
![EMTiming0_66](https://user-images.githubusercontent.com/38952746/152178868-dec1c6a0-aea7-4538-91ed-adea5dcefeb5.png)

Ignoring the first measurement, which is always higher, loading the large model takes 23% more time on average. While the difference may not sound extremely big, it is noticeable in feeling when switching between models in a user interface (for example in VSeeFace's avatar selection screen, which loads models for a preview when clicking on them).

**Comment by ousttrue** [comment]
Thank you for reporting.
`v0.90.0` https://github.com/vrm-c/UniVRM/pull/1373  may be relevant. 
Investigate the cause.

**Comment by ousttrue** [comment]
I checked the speed log with the importer. (Symbol  `VRM_DEVELOP`)
I tried the vivi model of vroid.

| version | first ReadMesh |
|---------|----------|
| v0.66.0 | 15ms     |
| v0.78.0 | 17ms     |
| v0.79.0 | 39ms   👀   |
| v0.89.0 | 47ms     |
| v0.90.0 | 42ms     |

It turns out that v0.79 slows down.
investigating···

**Comment by ousttrue** [comment]
The above measurements were inaccurate and did not help much.
I observed ReadMesh in the profiler.

![gc](https://user-images.githubusercontent.com/68057/155103503-13603467-1098-4227-a8a8-aeea4179d1f7.jpg)

It is caused in `v0.90.0`., and fixed it in #1535.

I think it has improved. Please report any problems.

`v0.96.0`

**Comment by emilianavt** [comment]
Thank you! The change definitely improves the loading times.
